tests/ports/esp32/ssl_dtls.py

```diff
@@ -0,0 +1,11 @@
+# Very basic test of ssl.SSLContext DTLS.
+
+try:
+    import ssl
```

Comment:
Minor, but so this test doesn't fail on all ports without DTLS support then need to also check for a DTLS name here (i.e. `ssl.PROTOCOL_DTLS_CLIENT` and catch `NameError` below to skip.